### DISCUSSION TITLE
Looking to help!!

### DISCUSSION THREAD

**Discussion by LCKidsLabs-01** [BIPES/BIPES#188] [General]
I did make a PR yesterday but I have many more things that I would like to be add or updated for a RP2040 (PI Pico, PI Pico, and Many others.) 

So my questions are: 

1)Could someone tell me what GitHub branch is running on the site "https://bipes.net.br/ide/"?

2)If the branch "https://bipes.net.br/ide/" is not the newest (vNext) version what branch is?

3)Do you have a style guide of any requirements for coding?

**Comment by jetannenbaum** [comment]
I don't know who is supporting the https://bipes.net.br/ide branch currently.  I'm supporting the offline branch, which is centered on support for the RP2040 boards.  It is currently a revision or two behind the https://bipes.net.br/pico/ui/ version.  I'll sync them up in the next couple of days.  Thank you for your support.
